cdap-data-fabric/src/main/java/co/cask/cdap/data2/transaction/queue/hbase/HBaseQueueAdmin.java

```diff
@@ -236,7 +234,9 @@ public void drop(String name) throws Exception {
 
   @Override
   public void upgrade(String tableName, Properties properties) throws Exception {
```

Comment:
I  think it was inherited from DatasetAdmin. Since now the QueueAdmin no longer extends from DatasetAdmin, I think can be changed to private.